src/main/java/com/powsybl/openloadflow/network/impl/LfVscConverterStationImpl.java

```diff
@@ -25,7 +25,7 @@ private LfVscConverterStationImpl(VscConverterStation station, boolean breakers,
 
         // local control only
         if (station.isVoltageRegulatorOn()) {
-            setVoltageControl(station.getVoltageSetpoint(), station.getTerminal(), breakers, report);
+            setVoltageControl(station.getVoltageSetpoint(), station.getRegulatingTerminal(), breakers, report);
```

Comment:
Can you add a test that shows that you are able to control voltage at a different terminal than the local one?